ovirt/resource_ovirt_vm.go

```diff
@@ -1609,3 +1631,48 @@ func getTemplateDiskAttachments(templateID string, meta interface{}) ([]*ovirtsd
 	}
 	return nil, nil
 }
+
+func getAffinityGroups(conn *ovirtsdk4.Connection, cID string, agNames []interface{}) (ag []*ovirtsdk4.AffinityGroup, err error) {
```

Comment:
Why is `agNames` an `[]interface{}`? It seems to me that there is no reason for it to be that way, except for the way you call it. Maybe casting it to `[]string` should be done in a separate helper function?

Reply:
Ammm we can have a helper function to convert it into a string array.
I guess that the proper way is to have the function receive what it needs to receive and not change the implementation due to the constraint just because it is more convenient... thanks for the feedback :)